recipes/typeguard/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "typeguard" %}
+{% set version = "2.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b8ddc6e2e60bd64b7003f9a685a09ba387b74adf2f6bea7534a76d61892f573e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python>=3.4
+    - pip
+    - setuptools_scm
+  run:
+    - python>=3.4
```

Comment:
you want a space here between python and >=3.4

Reply:
Is this just style, or? (I ask because last time I tried this it blew up on Windows, IIRC, and I'd rather stick to noarch).